src/generators/essudpgen/ReadoutGeneratorBase.h

```diff
@@ -9,6 +9,7 @@
 
 #pragma once
 
+#include "App.hpp"
```

Comment:
Where does this include come from?

Reply:
For CLI11 stuff normally
#include <CLI/CLI.hpp>
is enough. the package is being included from conan

Reply:
Thanks I see, I included the CLI.hpp instead.